keyboards/gami_studio/lex60/lex60.c

```diff
@@ -0,0 +1,13 @@
+#include "lex60.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	matrix_init_user();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
```

Comment:
These functions can be removed; they don't do anything the core implementation already does.